internal/controller/scheduler/scheduler.go

```diff
@@ -413,7 +451,6 @@ func (w *jobWrapper) Build() (*batchv1.Job, error) {
 		Command:         []string{"cp"},
 		Args: []string{
 			"/usr/local/bin/buildkite-agent",
-			"/usr/local/bin/ssh-env-config.sh",
```

Comment:
Undelete this

Reply:
We're still 0.x, so I think it's ok to break for someone who relies on it. I think it's unlikely anyone would have relied on it as:
1. The role of this script is to write out the contents of an environment variable as a `.ssh/id_x` file (or equivalent).
2. That was already done in the checkout container, and the result was copied into the `/workspace/.ssh` directory, so there was no need to run this script in the previous versions.
3. Indeed, the script was not run in the job container as we defined them

If a customer does want to run this script in their job container, our recommendation will be to get if from the source: https://github.com/buildkite/docker-ssh-env-config